packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -411,7 +413,11 @@ export default {
 
         await deployGatewayName(grid, selectionDetails.value!.domain, gwConfig);
         suggestName();
-        layout.value.setStatus("success", "Successfully deployed gateway.");
+        // get gateway url
+        await loadGateways();
```

Comment:
Nice! I have a suggestion: what about constructing the URL from the data we already have?